test/integration/consul-container/libs/assert/service.go

```diff
@@ -64,11 +66,30 @@ func HTTPServiceEchoes(t *testing.T, ip string, port int, path string) {
 		}
 
 		if !strings.Contains(string(body), phrase) {
-			r.Fatal("received an incorrect response ", body)
+			r.Fatal("received an incorrect response ", string(body))
 		}
 	})
 }
 
+// AssertServiceProxyAdminStatus validates all service subsets are up and running.
+func AssertServiceProxyAdminStatus(t *testing.T, ip string, port int) {
```

Comment:
How about "AssertServiceProxyAdminStatus validates envoy proxy returns statusOK"?

I don't think this `AssertServiceProxyAdminStatus` is necessary since GetEnvoyOutput would return the results with the status code.